NuRadioReco/detector/antennapattern.py

```diff
@@ -1172,7 +1195,30 @@ def get_antenna_response_vectorized(self, freq, zenith, azimuth, orientation_the
 
 class AntennaPattern(AntennaPatternBase):
     """
-    utility class that handles access and buffering to simulated antenna pattern
+    Utility class that handles access and buffering to simulated antenna pattern.
+    The class accesses the NuRadioReco pickle format file which contains the preprocessed antenna pattern.
+
+    The pickle file contains 9 lists of the following elements:
+
+    orientation_theta: float
+        orientation of the antenna, as a zenith angle (0deg is the zenith, 180deg is straight down); for LPDA: outward along boresight; for dipoles: upward along axis of azimuthal symmetry
+    orientation_phi: float
+        orientation of the antenna, as an azimuth angle (counting from East counterclockwise); for LPDA: outward along boresight; for dipoles: upward along axis of azimuthal symmetry
+    rotation_theta: float
+        rotation of the antenna, is perpendicular to 'orientation', for LPDAs: vector perpendicular to the plane containing the the tines
+    rotation_phi: float
+        rotation of the antenna, is perpendicular to 'orientation', for LPDAs: vector perpendicular to the plane containing the the tines
+    ff: array of floats
+        array of frequencies
+    thetas: array of floats
+        zenith angle of inicdent electric field
+    phis: array of floats
+        azimuth angle of incident electric field
+    H_phi: array of floats
+        the complex realized vector effective length of the ePhi polarization component
+    H_theta: array of floats
+        the complex realized vector effective length of the eTheta polarization component
```

Comment:
I have no idea how this will be formatted, I'll assume you checked - if it looks very messy maybe making it a list as in the other two places this occurs would look better.